schemas/lists/units.json

```diff
@@ -5,144 +5,236 @@
-      "pressure-millimeter-of-mercury",
-      "pressure-pound-per-square-inch",
+      "pressure-atmosphere",
+      "pressure-pascal",
+      "pressure-hectopascal",
+      "pressure-kilopascal",
+      "pressure-megapascal",
       "speed-kilometer-per-hour",
-      "speed-knot",
       "speed-meter-per-second",
       "speed-mile-per-hour",
+      "speed-knot",
+      "temperature-generic",
       "temperature-celsius",
       "temperature-fahrenheit",
-      "temperature-generic",
       "temperature-kelvin",
-      "volume-acre-foot",
-      "volume-bushel",
-      "volume-centiliter",
-      "volume-cubic-centimeter",
-      "volume-cubic-foot",
-      "volume-cubic-inch",
+      "torque-pound-force-foot",
+      "torque-newton-meter",
       "volume-cubic-kilometer",
       "volume-cubic-meter",
+      "volume-cubic-centimeter",
       "volume-cubic-mile",
       "volume-cubic-yard",
-      "volume-cup",
-      "volume-cup-metric",
-      "volume-deciliter",
-      "volume-fluid-ounce",
-      "volume-gallon",
-      "volume-gallon-imperial",
+      "volume-cubic-foot",
+      "volume-cubic-inch",
+      "volume-megaliter",
       "volume-hectoliter",
       "volume-liter",
-      "volume-megaliter",
+      "volume-deciliter",
+      "volume-centiliter",
       "volume-milliliter",
-      "volume-pint",
       "volume-pint-metric",
+      "volume-cup-metric",
+      "volume-acre-foot",
+      "volume-bushel",
+      "volume-gallon",
+      "volume-gallon-imperial",
       "volume-quart",
+      "volume-pint",
+      "volume-cup",
+      "volume-fluid-ounce",
+      "volume-fluid-ounce-imperial",
       "volume-tablespoon",
-      "volume-teaspoon"
+      "volume-teaspoon",
+      "volume-barrel",
+      "volume-dessert-spoon",
+      "volume-dessert-spoon-imperial",
+      "volume-drop",
+      "volume-dram",
+      "volume-jigger",
+      "volume-pinch",
+      "volume-quart-imperial",
+      "angle-steradian",
+      "concentr-portion",
+      "concentr-ofglucose",
+      "concentr-katal",
+      "duration-fortnight",
+      "electric-coulomb",
+      "electric-farad",
+      "electric-henry",
+      "electric-siemens",
+      "energy-calorie-it",
+      "energy-british-thermal-unit-it",
+      "energy-becquerel",
+      "energy-sievert",
+      "energy-gray",
+      "force-kilogram-force",
+      "length-100-kilometer",
+      "length-rod",
+      "length-chain",
+      "magnetic-tesla",
+      "magnetic-weber",
+      "mass-slug",
+      "pressure-ofhg",
+      "speed-beaufort",
+      "temperature-rankine",
+      "volume-pint-imperial",
+      "pressure-gasoline-equivalent",
+      "length-rin",
+      "length-sun",
+      "length-shaku-length",
+      "length-shaku-cloth",
+      "length-ken",
+      "length-jo-jp",
+      "length-ri-jp",
+      "area-bu-jp",
+      "area-se-jp",
+      "area-cho",
+      "volume-kosaji",
+      "volume-osaji",
+      "volume-cup-jp",
+      "volume-shaku",
+      "volume-sai",
```

Comment:
some of these aren't supported for en_GB - that something we need to deal with?

Reply:
One for @MebinAbraham, I guess.

Reply:
For now, we don't have a hard list of locale we allow or don't allow. Therefore, happy to keep this with the caveat that, it is on author/upstream to ensure locale and the units used lined up. If runner can technically support it, then we can keep it.